csharp/src/Ice/Communicator-ObjectAdapterFactory.cs

```diff
@@ -108,7 +108,7 @@ public async Task WaitForShutdownAsync()
             bool serializeDispatch = false,
             TaskScheduler? taskScheduler = null,
             Protocol protocol = Protocol.Ice2) =>
-            AddObjectAdapter(serializeDispatch: serializeDispatch, taskScheduler: taskScheduler, protocol: protocol);
+            AddObjectAdapter(serializeDispatch, taskScheduler, protocol);
```

Comment:
I would simply implement it right there, instead of calling a private method called only once!